src/main/java/de/mossgrabers/convertwithmoss/format/yamaha/ysfc/YamahaMOXFEWFMEntry.java

```diff
@@ -0,0 +1,37 @@
+package de.mossgrabers.convertwithmoss.format.yamaha.ysfc;
```

Comment:
To not pollute the main code with changes to this block, the entry for the section was subclassed. 

Not an ideal solution from theoretical point of view, but works.